pcs/src/traits.rs

```diff
@@ -0,0 +1,63 @@
+use arith::{Field, FieldSerde};
+use rand::RngCore;
+use std::fmt::Debug;
+
+pub trait PCS<F: Field + FieldSerde> {
+    type Params: Clone + Debug;
+    type Poly: Clone + Debug;
+    type EvalPoint: Clone + Debug;
+
+    type SRS: Clone + Debug + FieldSerde;
+    type PKey: Clone + Debug + From<Self::SRS> + FieldSerde;
+    type VKey: Clone + Debug + From<Self::SRS> + FieldSerde;
+    type Commitment: Clone + Debug + FieldSerde;
+    type Opening: Clone + Debug + FieldSerde;
+
+    fn gen_srs_for_testing(&self, rng: impl RngCore, params: &Self::Params) -> Self::SRS;
```

Comment:
propose narrower name like `gen_crs`

Reply:
oh, I guess you are copying from hyperplonk's pcs trait that this crs should not be used in production

Reply:
For the name, Yes. And I think `srs` is actually more frequently used than `crs` in latest literature.